vizro-core/examples/scratch_dev/data.yaml

```diff
@@ -0,0 +1,13 @@
+# Choose from 0-50
+setosa: 5
+versicolor: 10
+virginica: 15
+
+# Choose from: 4.8 to 7.4
+min: 5
+max: 7
+
+# Choose from:
+#   2020-01-01 to 2020-05-29
+date_min: 2024-01-01
+date_max: 2024-05-29
```

Comment:
Adjusting these values, the input data is changed for the `scratch/app.py` example.

Reply:
By adjusting `setosa`, `versicolor` and `virginica` (from 0 to 50), example graphs that use the `load_from_file_species` will be affected (page_1, page_2, page_5). 

By adjusting `min`, `max` (from 4.8 to 7.4), example graphs that use the `load_from_file_sepal_length` will be affected (page_3).

`TODO:` The following still doesn't work so enable page_4 and vm.DatePicker to support dynamic mode. 
By adjusting `date_min`, `date_max` (from 2020-01-01 to 2020-05-29), example graphs that use the `load_from_file_date_column ` will be affected (page_4).